Attempt at defining initialization, and simulation. (#3146)
modelica/ModelicaSpecification: chapters/classes.tex

\subsection{Component Variability Prefixes}\label{component-variability-prefixes-discrete-parameter-constant}

The prefixes \lstinline!discrete!, \lstinline!parameter!, \lstinline!constant! of a component declaration are called \firstuse[variability!prefix]{variability prefixes}\index{component variability}\index{declared variability}\index{variability!declared|see{declared variability}} and define in which situation the variable values of a component are initialized (see \cref{events-and-synchronization} and \cref{initialization-initial-equation-and-initial-algorithm}) and when they are changed during simulation:
\begin{itemize}
\item
  A variable \lstinline!vc! declared with \lstinline!constant!\indexinline{constant} prefix remains constant during simulation, with a value that is unaffected even by the initialization problem.
  This is called a \firstuse[---]{constant}, or \firstuse[constant!variable]{constant variable}\index{component variability!constant}.
\item

>   A variable \lstinline!vc! declared with the \lstinline!parameter!\indexinline{parameter} prefix has a value determined by the initialization problem, and keeps that value during the entire simulation.

Reviewer: We could reduce use of the lesser defined _initialization problem_ in favor of the defined _initialization_:
```suggestion
  A variable \lstinline!vc! declared with the \lstinline!parameter!\indexinline{parameter} prefix has a value determined at initialization, and keeps that value during the entire simulation.
```

\subsection{Component Variability Prefixes}\label{component-variability-prefixes-discrete-parameter-constant}

> The prefixes \lstinline!discrete!, \lstinline!parameter!, \lstinline!constant! of a component declaration are called \firstuse[variability!prefix]{variability prefixes}\index{component variability}\index{declared variability}\index{variability!declared|see{declared variability}} and define in which situation the variable values of a component are initialized (see \cref{events-and-synchronization} and \cref{initialization-initial-equation-and-initial-algorithm}) and when they are changed during the simulation:

Author: Or 'a simulation'?

> The prefixes \lstinline!discrete!, \lstinline!parameter!, \lstinline!constant! of a component declaration are called \firstuse[variability!prefix]{variability prefixes}\index{component variability}\index{declared variability}\index{variability!declared|see{declared variability}} and define in which situation the variable values of a component are initialized (see \cref{events-and-synchronization} and \cref{initialization-initial-equation-and-initial-algorithm}) and when they are changed during the simulation:

Reviewer: I prefer the symmetry with phrases such as `during flattening`,  `during translation`, or `during (simulation) initialization`.
